src/BucketListItem.jsx

```diff
@@ -1,5 +1,23 @@
+import { createEffect, createSignal } from 'solid-js';
+import {Modal} from './Modal';
+
 export function BucketListItem(props) {
+  const [isOpen, setIsOpen] = createSignal(false);
+  
+  const handleDeleteItem = ()=>{
+      props.setItems((items) => {
```

Comment:
As the issue talks about deleting one or more wishes.. it is better to add a delete link at the top of the wish list.. it will help us to select one or more wishes and then click on the single delete button.

the modal looks good.

Reply:
Apologies for not being able to implement the desired feature initially. Could you please confirm if I understand correctly? You're suggesting adding a delete button at the top of the wishlist. Users can then select multiple items and delete them with a single click. If my interpretation is incorrect, please provide clarification. Thank you for your feedback.

Reply:
Sir @atapas, I've implemented the feature. Please check the attached video. If it looks good, I'll proceed with the changes. Any feedback is welcome.

https://github.com/tapascript/solid-bucket-list/assets/84084280/6c967c59-e4ac-445d-a36d-636925029ac2